### PR TITLE
Add cloudformationSchema util

### PR DESCRIPTION
**Pull request by fredericbarthelet**
Addresses: https://github.com/serverless/serverless/issues/8627

### CODE REVIEW

File: cloudformation-schema.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const createSchema = () => {
+  const types = _.flatten(
+    functionNames.map((functionName) =>
+      ['mapping', 'scalar', 'sequence'].map((kind) => yamlType(functionName, kind))
+    )
+  );
+  return YAML.Schema.create(types);
+};
+
+module.exports = {
```
Comment (medikoo) [inline cloudformation-schema.js:52]: Let's export schema directly on `module.exports` (or do we have some specific reason to create here additional namespace?)

**Review by medikoo — CHANGES_REQUESTED** [review]
Thank you @fredericbarthelet looks great! I have just one minor suggestion

File: cloudformation-schema.js (hunk 1 above)
Comment (fredericbarthelet) [reply cloudformation-schema.js]: No particular reason other than keeping the original file. I updated to directly export the schema

**Review by medikoo — APPROVED** [review]
Thank you @fredericbarthelet !

**Pull request merged**